generate/post_process.go

```diff
@@ -0,0 +1,39 @@
+package generate
+
+import (
+	"strconv"
+
+	"github.com/gunk/gunk/config"
+)
+
+func parseJSONTagPostprocParam(gen config.Generator) (bool, error) {
+	const jsonTagPostprocParamName = "json_tag_postproc"
+	for _, param := range gen.Params {
+		if param.Key == jsonTagPostprocParamName {
```

Comment:
do we really not have a func to do this already? we have multiple of these boolean flags in the config file.